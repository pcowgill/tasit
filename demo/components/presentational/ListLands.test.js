import React from "react";
import { shallow } from "enzyme";
import ListLands from "./ListLands";

describe("ListLands", () => {
  it("renders the component", async () => {
    expect(
      shallow(<ListLands dataSource={{}} renderRow={() => {}} />)
    ).toMatchSnapshot();
  });
});
