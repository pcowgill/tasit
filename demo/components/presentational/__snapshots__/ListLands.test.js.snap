// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListLands renders the component 1`] = `
<ListViewMock
  dataSource={Object {}}
  renderRow={[Function]}
  renderScrollComponent={[MockFunction]}
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
/>
`;
